deps/verifier/src/intel_dcap/error.rs

```diff
@@ -0,0 +1,80 @@
+use intel_tee_quote_verification_rs::quote3_error_t;
+
+/// List of DCAP related errors.
+/// <https://download.01.org/intel-sgx/sgx-dcap/1.23/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf>
+pub fn describe_error(value: quote3_error_t) -> String {
+    let name = match value {
+        quote3_error_t::SGX_QL_ERROR_UNEXPECTED => "An unexpected internal error occurred",
+        quote3_error_t::SGX_QL_ERROR_INVALID_PARAMETER => "The platform quote provider library rejected the input.",
+        quote3_error_t::SGX_QL_ERROR_OUT_OF_MEMORY => "Heap memory allocation error in library or enclave.",
+        quote3_error_t::SGX_QL_FILE_ACCESS_ERROR => "Not able to find the ‘label’ or there was a problem writing or retrieving the data.",
+        quote3_error_t::SGX_QL_UNSUPPORTED_ATT_KEY_ID => "Unsupported attestation key ID.",
+        quote3_error_t::SGX_QL_UNSUPPORTED_LOADING_POLICY => "Selected policy is not supported by the quoting library.",
+        quote3_error_t::SGX_QL_PLATFORM_LIB_UNAVAILABLE => "The Quote Verification Library could not locate the provider library.",
+        quote3_error_t::SGX_QL_ATT_KEY_NOT_INITIALIZED => "Platform quoting infrastructure does not have the attestation key available to generate quotes.",
+        quote3_error_t::SGX_QL_ATT_KEY_CERT_DATA_INVALID => "Certification data retrieved from the platform quote provider library is invalid",
+        quote3_error_t::SGX_QL_NO_PLATFORM_CERT_DATA => "The platform quote provider library doesn't have the platform certification data for this platform.",
+        quote3_error_t::SGX_QL_ERROR_REPORT => "The QvE report can NOT be verified.",
+        quote3_error_t::SGX_QL_ENCLAVE_LOST => "Enclave was lost after power transition or used in a child process created by linux:fork()",
+        quote3_error_t::SGX_QL_INVALID_REPORT => "Report MAC check failed on an application report.",
+        quote3_error_t::SGX_QL_ENCLAVE_LOAD_ERROR => "Unable to load one of the quote library enclaves required to initialize the attestation key. Could be due to file I/O error or some other loading infrastructure errors.",
+        quote3_error_t::SGX_QL_NETWORK_ERROR => "If the platform quote provider library uses the network to retrieve the QVE Identity, this error will be returned when it encounters network connectivity problems.",
```

Comment:
most often we have users with `0xe019` error and I think it's this one. Trustee uses the default QPL so could we give a hint this is about `sgx_default_qcnl.conf` configuration?